AsyncSwift/Observed/AppData.swift

```diff
@@ -13,14 +13,19 @@ final class AppData: ObservableObject {
     @Published var currentTab: Tab = .event
     
     private var currentStamp: Stamp?
-    lazy var isStampExist: Bool = {
-        KeyChain.shared.getItem(key: currentStamp?.title) != nil
-    }()
     
     init(){
         fetchCurrentStamp()
     }
 
+    func isStampExist() -> Bool {
```

Comment:
```suggestion
    var isStampExist: Bool {
```

바로 return 해줘도 되지 않을까 합니다 .. !